website/content/docs/authentication-backends.md

```diff
@@ -27,6 +27,10 @@ To use it in your own project stored on GitHub or GitLab, follow these steps:
 
 3. Optionally, you can assign roles to users in your Netlify dashboard, and then limit which roles can access the CMS by defining the `accept_roles` field as shown in the example above. Otherwise `accept_roles` can be left out, and all Netlify Identity users on your site have access.
 
+### Reconnect after Changing Repository Permissions
+
+If you change ownership on your repository, or convert a repository from public to private, you may need to reconnect Git Gateway with proper permissions. See [Reconnect after Changing Repository Permissions](https://www.netlify.com/docs/git-gateway/#reconnect-after-changing-repository-permissions) for instructions.
```

Comment:
I think linking to the instructions makes perfect sense. I think I would rename the link to make it clear that the reader will be sent elsewhere, since the "Reconnect after..." title is identical to the current section. For example, "Find further instructions in the [Netlify docs].

Reply:
👍 good point, fixed! 

Do you prefer `[Netlify Git Gateway docs]` or `[Netlify docs]`? 

I went with the former to match `[Netflix Identity docs]`, but then noticed just `[Netlify docs]` further down the page...